Core/Src/main.c

```diff
@@ -90,16 +119,34 @@ int main(void)
   MX_GPIO_Init();
   MX_TIM3_Init();
   MX_TIM2_Init();
+  MX_I2C1_Init();
+  MX_USART2_UART_Init();
   /* USER CODE BEGIN 2 */
   L298N_Init();
+  HMC5883L_Init();
   /* USER CODE END 2 */
 
   /* Infinite loop */
   /* USER CODE BEGIN WHILE */
   L298NDriver_SetSpeed(&bridge_handler, 0.5);
   L298NDriver_MoveForward(&bridge_handler);
+
+//  HMC5883L_Calibrate();
+  compass_calibration_params.x_offset = 24;
+  compass_calibration_params.y_offset = 14;
+  compass_calibration_params.x_scale = 1.0;
+  compass_calibration_params.y_scale = (float)138/135;
```

Comment:
These are default values? Why not placing them inside the `CompassService_InitCalibrationParams` function?

Reply:
`CompassService_InitCalibrationParams` function setup the structure parameters to start a new calibration... These values ​​that are there were obtained when the current hardware was calibrated, and will change from different hardwares.